docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -26,7 +26,7 @@ Even when the underlying storage bucket is the same, backups from one are not co
 
 ### S3 backup store
 
-To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket:
+To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket. This configuration can be set in your Zeebe [`config/application.yaml`](/docs//self-managed/zeebe-deployment/configuration/configuration.md):
```

Comment:
Is the double slash intended? `/docs//`

Reply:
Thanks for catching this. updated to fix 👍